### PR TITLE
Update sos dockerfile prereqs to install sgx 18.04 packages

### PR DESCRIPTION
**Pull request by vtikoo**

### CODE REVIEW

File: tests/dotnet-sos/Dockerfile-prereqs [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN dotnet tool install -g dotnet-sos \
 	&& /root/.dotnet/tools/dotnet-sos install
 
 # Install intel library required to run enclaves
-RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu xenial main' | tee /etc/apt/sources.list.d/intel-sgx.list && \
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | tee /etc/apt/sources.list.d/intel-sgx.list && \
```
Comment (bodzhang) [inline tests/dotnet-sos/Dockerfile-prereqs:16]: Is the base image on line 1 Ubuntu18.04 based?

Comment (vtikoo) [reply tests/dotnet-sos/Dockerfile-prereqs:16]: @bodzhang the image is debian 10(buster) based. 
```
$ docker run --rm mcr.microsoft.com/dotnet/sdk:5.0 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
```

Comment (bodzhang) [reply tests/dotnet-sos/Dockerfile-prereqs:16]: Hmm, Intel's PSW doe not officially support Debian.

Comment (vtikoo) [reply tests/dotnet-sos/Dockerfile-prereqs:16]: dotnet doesn't support ubuntu 18.04 in its 5.0 release. There is a image available for ubuntu 20.04 however.
https://hub.docker.com/_/microsoft-dotnet-sdk

Comment (bodzhang) [reply tests/dotnet-sos/Dockerfile-prereqs:16]: Even if Intel's PSW does not officially support Debian, but if the Ubuntu bionic package from Intel works in this Debian based container image, I think it's OK for the test case.

**Review by bodzhang — APPROVED** [review]

**Pull request merged**